default.json

```diff
@@ -54,7 +54,7 @@
       "groupName": "npm definitely typed",
       "prPriority": 99,
       "recreateClosed": true,
-      "schedule": "before 3:00 am on Tuesday"
+      "schedule": "before 3:00 am every 2 weeks on Tuesday"
```

Comment:
Is it worth trying to offset these so they aren't the same Tuesday/Wednesday? It looks like there could be some horrific incantation that would unleash that (e.g. "before 3:00 am every 2 weeks on Tuesday starting on the 7th day")

Reply:
Which ones were you hoping to stagger? The intent is for the DefinitelyTyped PRs to be automerged before the `devDependencies` come out to play, and those two tend to be the most noisy groups.

I feel like biting the bullet one week then getting the next one off (except for SEEK packages 🙂) may be a better dev experience.